cli/viam/main.go

```diff
@@ -174,21 +178,25 @@ func main() {
 							if err != nil {
 								return err
 							}
-							for _, org := range orgs {
-								fmt.Fprintf(c.App.Writer, "%s (id: %s)\n", org.Name, org.Id)
+							for i, org := range orgs {
+								if i == 0 {
+									fmt.Fprintf(c.App.Writer, "Organizations for %q:\n", client.Config().Auth.User.Email)
+								}
+								fmt.Fprintf(c.App.Writer, "\t%s (id: %s)\n", org.Name, org.Id)
 							}
 							return nil
 						},
 					},
 				},
 			},
 			{
-				Name:  "locations",
-				Usage: "work with locations",
+				Name:            "locations",
+				Usage:           "work with locations",
+				HideHelpCommand: true,
 				Subcommands: []*cli.Command{
 					{
 						Name:      "list",
-						Usage:     "list locations",
+						Usage:     "list locations for the current user",
 						ArgsUsage: "[organization]",
```

Comment:
You seem to have removed ArgsUsage elsewhere in favor of UsageText. Did you intentionally keep it here?

Reply:
Yeah I thought I'd keep it; this help message seems fine:
```
=> viam locations list --help
NAME:
   viam locations list - list locations for the current user

USAGE:
   viam locations list [command options] [organization]

OPTIONS:
   --help, -h  show help (default: false)
```